index.js

```diff
@@ -68,8 +68,7 @@ function loader (registryOrVersion) {
     }
 
     static toNotch (item, serverAuthoritative = true) {
-      const hasNBT = item && item.nbt && Object.keys(item.nbt.value).length > 0
-
+      const hasNBT = !!(item && item.nbt && Object.keys(item.nbt.value).length > 0)
```

Comment:
Why?

Reply:
hasNBT is null instead of false when item.nbt is null

Reply:
Ok, is there a reason it can't be fasleish? Seems implicative its necessary here.

Reply:
> Ok, is there a reason it can't be fasleish? Seems implicative its necessary here.

Mostly just laziness.. Correcting all the hardcoded hasNBT: false in the tests takes much longer than just converting to boolean here

Reply:
Ok, can you use Boolean() here? I think it would be clearer.